app/javascript/utils/form/final_form/MultiSelectField.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react'
+
+import classNames from 'classnames'
+import PropTypes from 'prop-types'
+import { Field } from 'react-final-form'
+
+import MultiSelect from 'components/molecules/MultiSelect'
+
+import { FieldError, HelpText, Label } from './Field'
+
+const format = (options) => {
+  return options && options.map((option) => (typeof option === 'string' ? option : option.value))
+}
+const HEIGHT_NORMAL = 'normal'
```

Comment:
This might be irrelevent depending on if you remove the height values, but can you export the consts from `MultiSelect.js` and use the imported versions here? since they need to be in sync.